src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_plugin/core_integration.cpp

```diff
@@ -123,6 +124,200 @@ INSTANTIATE_TEST_SUITE_P(nightly_OVClassGetMetricTest,
         OVClassGetMetricTest_GPU_EXECUTION_UNITS_COUNT,
         ::testing::Values("GPU"));
 
+
+using OVClassGetPropertyTest_GPU = OVClassBaseTestP;
+TEST_P(OVClassGetPropertyTest_GPU, GetMetricSupporetedPropertiesAndPrintNoThrow) {
+    ov::Core ie;
+    ov::Any p;
+
+    ASSERT_NO_THROW(p = ie.get_property(deviceName, ov::supported_properties));
+    std::vector<ov::PropertyName> t = p;
+
+    std::cout << "SUPPORTED_PROPERTIES: ";
+    for (const auto& prop : t) {
+        std::cout << prop << " ";
+    }
+    std::cout << std::endl;
+
+    OV_ASSERT_PROPERTY_SUPPORTED(ov::supported_properties);
+}
+
+TEST_P(OVClassGetPropertyTest_GPU, GetMetricAvailableDevicesAndPrintNoThrow) {
+    ov::Core ie;
+    ov::Any p;
+
+    ASSERT_NO_THROW(p = ie.get_property(deviceName, ov::available_devices));
```

Comment:
`ie.get_property(deviceName, ov::available_devices)` call returns `std::vector<std::string>` so you don't need `ov::Any` here

Reply:
Please remove `ov::Any` from all other tests here if they used `ov::Property` variables

Reply:
Fixed, thanks